apiserver/cache/redis.go

```diff
@@ -89,6 +90,19 @@ func (r *Cache) RemoveUserToken(ctx goContext.Context, token string) error {
 	return nil
 }
 
+func jtiKey(userId int, jti string) string {
+	return fmt.Sprintf("jti-%v-%v", strconv.Itoa(userId), jti)
```

Comment:
Let's use `:` as separator for Redis keys

e.g. `fmt.Sprintf("userId:%d:jti:%s", userId, jti)`
also pls update `userIdKey` to 
```
func userIdKey(token string) string {
	return "sessionToken:" + token
}
```